### ISSUE TITLE
Use: modrinth.com/mod/smooth-hud

### ISSUE THREAD

**Issue by ghost** [polina4096/butter#8]
port to 1.21, fabric? Also can you upload it to modrinth?

**Comment by polina4096** [comment]
It is on modrinth..? Where did you get it?
https://modrinth.com/mod/butter

**Comment by polina4096** [comment]
Will update it soon ^_^

**Comment by ghost** [comment]
Any updates?

**Comment by molocko** [comment]
I'm really looking forward to the update

**Comment by MoupiPics** [comment]
use this as an alternative https://modrinth.com/mod/smooth-hud

**Comment by polina4096** [comment]
Yeah, use that. It's better and I'm not sure I'll be updating this mod any time soon.
